scripts/Tools/testreporter.py

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/env python
```

Comment:
Follow the template of other python scripts in cime.   Each script has a test to see if it was called directly
or from another python, and it has a parse_command_line subroutine which includes code for the standard cime options.   I think that create_clone provides a simple example.